README.md

```diff
@@ -19,29 +19,26 @@ guide](http://guides.cocoapods.org/using/getting-started.html#installation) and
 
 ## Project Goals
 
-Ultimately, the goal of CocoaPods is to improve the engagement in, and
-discoverability of, third party open-source libraries. The following list
-includes the project goals which have influenced, and continue to
-drive the design of CocoaPods.
-
-- Being able to work in the system without creating extra work for
-  library authors. Making it possible to maintain a simple transparent
-  podspec outside of the library’s repository.
-- Not imposing any judgement of ‘correctness’ on how authors decide to
-  structure their library.
-- CocoaPods should not impose any specific requirements on source-code
-  management systems. (Currently supported are git, svn, mercurial, bazaar, and
-  various types of archives downloaded over HTTP.)
-- Provide the ability to work in a distributed way, but also provide
+CocoaPods aims to improve the engagement with, and discoverability 
+of, third party open-source Cocoa libraries. These
+project goals influence and drive the design of CocoaPods:
+
+- Create and share libraries, and use them in your own projects,
+  without creating extra work for library authors. Hack on your own fork 
+  of any library with a simple transparent `Podspec` standard.
```

Comment:
This is now missing an entry about not needing a fork at all in order to be able to provide a podspec, which was more what the original goal tried to describe.